Source/WebInspectorUI/UserInterface/Views/Popover.css

```
@@ -34,8 +34,13 @@
     --popover-background-color: var(--panel-background-color);
     --popover-border-color: hsla(0, 0%, 0%, 0.25);
     --popover-shadow-color: hsla(0, 0%, 0%, 0.5);
+}
 
-    background-image: -webkit-canvas(popover);
+.popover.background-canvas {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index : -1;
```

```suggestion
    inset: 0;
    z-index: -1;
```
I think it makes sense to set bottom/right as well to 0. `inset` should set all 4 sides.